citest/json_predicate/binary_predicate.py

```diff
@@ -428,8 +458,19 @@ def __call__(self, value):
     '==', lambda a, b: a == b, operand_type=list)
 LIST_NE = StandardBinaryPredicateFactory(
     '!=', lambda a, b: a != b, operand_type=list)
+
+def lists_equivalent(a, b):
```

Comment:
I cannot use sets because the lists often contain dictionaries which are not hashable so cannot convert here. That's why I wrote my own function.